mmv1/third_party/terraform/.teamcity/components/projects/feature_branches/FEATURE-BRANCH-ephemeral-write-only.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// This file is maintained in the GoogleCloudPlatform/magic-modules repository and copied into the downstream provider repositories. Any changes to this file in the downstream will be overwritten.
+
+package projects.feature_branches
+
+import ProviderNameBeta
+import ProviderNameGa
+import SharedResourceNameBeta
+import SharedResourceNameGa
+import SharedResourceNameVcr
+import builds.*
+import generated.ServicesListBeta
+import generated.ServicesListGa
+import jetbrains.buildServer.configs.kotlin.Project
+import replaceCharsId
+import vcs_roots.HashiCorpVCSRootBeta
+import vcs_roots.HashiCorpVCSRootGa
+import vcs_roots.ModularMagicianVCSRootBeta
+import vcs_roots.ModularMagicianVCSRootGa
+import getServicesList
+
+const val featureBranchEphemeralWriteOnly = "FEATURE-BRANCH-ephemeral-write-only"
+const val EphemeralWriteOnlyTfCoreVersion = "1.11.0-alpha20241211"
+
+fun featureBranchEphemeralWriteOnlySubProject(allConfig: AllContextParameters): Project {
+
+    val trigger  = NightlyTriggerConfiguration(
+        branch = "refs/heads/$featureBranchEphemeralWriteOnly" // triggered builds must test the feature branch
+    )
```

Comment:
Please see other comment about conflicting with the nightly tests, particularly for Compute.

To mitigate this, you could update the trigger in the feature branch test project to run on a later schedule versus the nightlies. E.g. default time + 6 hours